### DISCUSSION TITLE
Imported photos turning from portrait to landscape if compressed

### DISCUSSION THREAD

**Discussion by Heidieker** [TriliumNext/Trilium#3274] [Q&A]
Hi, 
i recognized, that imported photos turning from portrait to landscape if compressed.
I am using the my mobile to take some notes and adding photos. When the photo is made in portrait-mode it will turn to landscape, if i check compression. 
Is there a workaround?

**Comment by zadam** [comment]
Hi, that sounds like a bug, but I can't reproduce it - for me the portrait images are imported as portrait.

Would you have an example of an image which does this? You can send it to me at zadam.apps@gmail.com

**Reply by zadam** [reply]
Thanks to @Heidieker for the provided image I was able to reproduce the problem. It seems like a problem in Jimp library (resp. for the resizing/compression) - I created a bug report here: https://github.com/oliver-moran/jimp/issues/1104
